base/pom.xml

```diff
@@ -88,6 +88,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>dockerfile-maven-plugin</artifactId>
+                <configuration>
+                    <buildArgs>
+                        <!-- Redhat UBI Minimal Version -->
+                        <UBI_MINIMAL_VERSION>8.3-298</UBI_MINIMAL_VERSION>
+                        <!-- Redhat Package Versions -->
+                        <OPENSSL_VERSION>-1.1.1g-15.el8_3</OPENSSL_VERSION>
```

Comment:
Should we upgrade to 1.1.1k or is this simply tracking the openssl version included in the based UBI image?

Reply:
This is the latest version of openssl that Redhat has built for their EL8 OS - Future point releases (ie: 8.4) of Redhat may bump this to 1.1.1k when they see fit to do a breaking change over a point release.

Most certainly we do not want to build any other version of openssl for the OS: Too many things depend on it.